operator/api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,113 @@
+// +build !ignore_autogenerated
+
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// autogenerated by controller-gen object, do not modify manually
+
+package v1beta1
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
+)
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *HelmRelease) DeepCopyInto(out *HelmRelease) {
```

Comment:
The auto generated deep copy methods here look good. However, it seems that some other generated methods are missing. For example, the methods found [here](https://github.com/Wattpad/ship-it/tree/master/pkg/generated) seem to be unavailable.